### PR TITLE
ADEN-6779 Enable Mega on BLB everywhere

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php [hunk 1]
```diff
@@ -120,6 +121,7 @@ public static function onInstantGlobalsGetNewsAndStoriesVariables( array &$vars
 		$vars[] = 'wgAdDriverVideoMoatTrackingCountries';
 		$vars[] = 'wgAdDriverVideoMoatTrackingSampling';
 		$vars[] = 'wgAdDriverVideoPostrollCountries';
+		$vars[] = 'wgAdDriverBottomLeaderBoardMegaCountries';
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: Alphabetical order please

File: extensions/wikia/AdEngine/js/context/adContext.js [hunk 2]
```diff
@@ -211,8 +211,8 @@ define('ext.wikia.adEngine.adContext', [
 		context.opts.replayAdsForFV = geo.isProperGeo(instantGlobals.wgAdDriverPlayAdsOnNextFVCountries);
 		context.opts.fvAdsFrequency = fvAdsFrequency !== undefined ? fvAdsFrequency : 3;
 		context.opts.disableSra = geo.isProperGeo(instantGlobals.wgAdDriverDisableSraCountries);
-		context.opts.isBLBOnMobileEnabled =
-			geo.isProperGeo(instantGlobals.wgAdDriverBottomLeaderBoardOnMobileCountries);
+		context.opts.isBLBOnMobileEnabled = geo.isProperGeo(instantGlobals.wgAdDriverBottomLeaderBoardOnMobileCountries);
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/context/adContext.js]: This line exceeds 120 chars :P

File: extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js [hunk 3]
```diff
@@ -9,7 +9,8 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 
 	var dfpId = '5441',
 		megaSlots = [
-			'INVISIBLE_SKIN'
+			'INVISIBLE_SKIN',
+			//ToDo: BLB Mega global - 'BOTTOM_LEADERBOARD'
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js]: Add ticket number please (create one for clean up if it doesn't exist)

File: extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js [hunk 4]
```diff
@@ -106,9 +107,9 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 	}
 
 	function isMegaSlot(slotName) {
-		return megaSlots.indexOf(slotName) !== -1 ||
-				// TODO move BLB to megaSlots in ADEN-6779
-			(getContextTargeting().skin === 'mercury' && slotName === 'BOTTOM_LEADERBOARD');
+		return slotName === 'BOTTOM_LEADERBOARD' ?
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js]: This if statement is getting bigger. What about:
```
if (context.opts.isBLBMegaEnabled) {
  megaSlots.push('BOTTOM_LEADERBOARD');
}

if (context.opts.isBLBOnMobileEnabled) {
  megaSlots.push('MOBILE_BOTTOM_LEADERBOARD');
}
```
somewhere above?

Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js]: Use `adContext.get('opts.foo')` please

Comment (kvas-damian) [reply extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js]: and please leave a TODO to cleanup after we fully roll out

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: Moved :>

File: extensions/wikia/AdEngine/js/context/adContext.js (hunk 2 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/context/adContext.js]: By one char :o Splited.

File: extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js (hunk 3 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js]: Done.

File: extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js (hunk 4 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js]: 3x Done.

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**